1

```diff
@@ -0,0 +1,21 @@
+Merge branch 'icx_kss_2' of https://github.com/jianlia/openenclave into icx_kss_2
```

Comment:
This should not be included as part of the PR. Please resolve the merge conflict and remove this file.

Reply:
Updated